Add predicates for all types

joda opened the pull request:
Provide inst?-like predicates for all cljc.java-time types.

joda committed:
Add predicates for all types

henryw374 commented:
bit of de ja vu https://github.com/henryw374/time-specs/blob/master/src/time_specs/core.cljc ;-)

ok I think this can go in this lib. the time-specs lib can use these preds if it ever gets written.

the only thing is that every other ns here just mirrors java.time until now. maybe it should be 'cljc.java-time.extn.predicate' package or something for clarity? and have the `extn` be a place to put any other stuff that's not come from java.time in future?

joda commented:
Haha :). Yeah, I wasn't sure this was the right place, but took a chance - it seemed like the best guess for now at least. Sure, I can put it in a different ns. It kind of felt wrong to put it next to the generated code.

joda committed:
Move predicates to extn ns.

Since everyting else is a mirror of java.time, put the predicates
in a separate ns to indicate that this is something else than
java.time. The extn ns will then also be a place to put other
extensions.

henryw374 commented:
cheers. done release now. and new tick release
